gapic/generator/generator.py

```diff
@@ -92,31 +90,30 @@ def get_response(
         # and plug those into the template.
         for template_name in client_templates:
             # Sanity check: Skip "private" templates.
-            filename = template_name.split('/')[-1]
-            if filename.startswith('_') and filename != '__init__.py.j2':
+            filename = template_name.split("/")[-1]
+            if filename.startswith("_") and filename != "__init__.py.j2":
                 continue
 
             # Append to the output files dictionary.
             output_files.update(
                 self._render_template(
-                    template_name,
-                    api_schema=api_schema,
-                    opts=opts
-                )
+                    template_name, api_schema=api_schema, opts=opts)
             )
 
-        output_files.update(self._generate_samples_and_manifest(
-            api_schema,
-            self._env.get_template(sample_templates[0]),
-        ))
+        output_files.update(
+            self._generate_samples_and_manifest(
+                api_schema, self._env.get_template(sample_templates[0]),
+            )
+        )
 
         # Return the CodeGeneratorResponse output.
-        return CodeGeneratorResponse(file=[i for i in output_files.values()])
+        res = CodeGeneratorResponse(
+            file=[i for i in output_files.values()])  # type: ignore
+        res.supported_features |= CodeGeneratorResponse.Feature.FEATURE_PROTO3_OPTIONAL  # type: ignore
```

Comment:
Just want to check that this is intentional (since the other changes in this file seem to be formatting)

Reply:
The formatting got fed through a log chipper due to (I think) a horrible interaction between internal Google formatting logic and custom opensource style formatting logic.
I've resolved the aggressive autoformating issue in emacs; this should hopefully be the last review where it is such an issue.